chromium_src/chrome/browser/ui/views/permissions/permission_prompt_bubble_base_view.cc

```diff
@@ -42,10 +48,24 @@
 #include "ui/views/window/dialog_client_view.h"
 #include "ui/views/window/dialog_delegate.h"
 
+#if BUILDFLAG(IS_WIN)
+#include "brave/browser/ui/geolocation/geolocation_utils_win.h"
+#endif
+
 #if BUILDFLAG(ENABLE_WIDEVINE)
 #include "brave/browser/widevine/widevine_permission_request.h"
 #endif
 
+constexpr char kLearnMoreURL[] =
+#if BUILDFLAG(IS_WIN)
+    "https://support.microsoft.com/en-us/windows/"
+    "windows-location-service-and-privacy-3a8eee0a-5b0b-dc07-eede-"
+    "2a5ca1c49088";
+#else
+    // TODO(simonhong): Use different url for other platforms.
```

Comment:
@rebron We need different learn more link for macOS or linux.

Reply:
Used https://support.apple.com/guide/mac-help/allow-apps-to-detect-the-location-of-your-mac-mh35873/mac for macOS
Used https://help.ubuntu.com/stable/ubuntu-help/privacy-location.html on linux. it's ubuntu specific link so not sure it's suitable for this learn more link.